src/loki/CMakeLists.txt

```diff
@@ -11,7 +11,8 @@ set(sources
   isochrone_action.cc
   trace_route_action.cc
   transit_available_action.cc
-  node_search.cc)
+  node_search.cc
+  polygon_search.cc)
```

Comment:
Since this is a new file, can you also add it to [`set_source_files_properties` below](https://github.com/valhalla/valhalla/pull/2750/files#diff-7552fd95e00220b1c45ebce4ff7c0c24e47586fa3feff85f77cbde5dcab65c79R20) so that we have stricter compiler checks enabled for it?

We're doing it piece-meal for existing files that we're making changes to (to reduce diff noise from just warning fixes)